models/post.js

```diff
@@ -0,0 +1,12 @@
+var mongoose=require('mongoose')
+var Schema=mongoose.Schema
+var schema = new Schema({
+    imagePath:{type:String,required:true},
+    name:{type:String,required:true},
+    location:{type:String,required:true},
+    date:{type : Date, default: Date.now},
+    message:{type:String,required:true},
```

Comment:
There must be a key to keep track of likes for a particular post.

Reply:
sir i added likes key in schema and default data sir.